### PR TITLE
Update all files with suggestions

### PR DESCRIPTION
**Pull request by musayuksel**

### CODE REVIEW

File: src/controllers/userController/createNewUser.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export const createNewUser = async (req: Request, res: Response) => {
+  const { userName, userEmail, firstName, lastName } = req.body;
+  try {
+    const user = await User.findOne({
+      where: {
+        userEmail,
+      },
+    });
+    if (user) {
+      console.log('user already exists>--------------------');
```
Comment (musayuksel) [inline src/controllers/userController/createNewUser.ts]: It doesn't send the correct error and the app crashes.
Delete console!

File: src/schemas/channel.schema.ts [hunk 2]
```diff
@@ -2,4 +2,24 @@ import * as Joi from 'joi';
 
 export const channelSchema = Joi.object({
   channelName: Joi.string().required(),
-});
\ No newline at end of file
+});
+
+export const messageSchema = Joi.object({
```
Comment (robertwilkinson-dev) [inline src/schemas/channel.schema.ts]: i'd personally spit these into their own files: e.g: message.schema.ts

File: src/routes/messagesRoutes.ts [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
+  '/:userId',
+  validateReqParamSchema(userIdParamSchema),
+  messageController.getUserMessages
+);
+router.post(
+  '/',
+  logBody,
+  validateReqBodySchema(messageSchema),
+  messageController.createMessage
+);
```
Comment (robertwilkinson-dev) [inline src/routes/messagesRoutes.ts]: maybe we should look at adding some lint/prettier rules? what do you think?

File: src/controllers/userController/createNewUser.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      throw new CustomError('User already exists', 400);
+    }
+
+    const user1 = await User.create({
```
Comment (robertwilkinson-dev) [inline src/controllers/userController/createNewUser.ts]: why is this "user1" and not just "user"?

File: src/controllers/userController/createNewUser.ts (hunk 1 above)
Comment (robertwilkinson-dev) [reply src/controllers/userController/createNewUser.ts]: it crashes?

File: src/controllers/userController/createNewUser.ts (hunk 4 above)
Comment (musayuksel) [reply src/controllers/userController/createNewUser.ts]: I was using it in the same scope to check if the user already exists. I have updated it.

File: src/routes/messagesRoutes.ts (hunk 3 above)
Comment (musayuksel) [reply src/routes/messagesRoutes.ts]: It would be great, I will try to do it.

File: src/schemas/channel.schema.ts (hunk 2 above)
Comment (musayuksel) [reply src/schemas/channel.schema.ts]: It's very makes sense. I have separated them.

File: src/controllers/userController/createNewUser.ts (hunk 1 above)
Comment (musayuksel) [reply src/controllers/userController/createNewUser.ts]: When I try to add the same user again to the DB( if user exist), the app crashes.

Comment (robertwilkinson-dev) [reply src/controllers/userController/createNewUser.ts]: addressed in slack message

**Pull request merged**